core/src/main/java/nl/nn/adapterframework/receivers/Receiver.java

```diff
@@ -1462,10 +1457,9 @@ private boolean isDuplicateAndSkip(IMessageBrowser<Serializable> transactionStor
 	@Override
 	public void exceptionThrown(INamedObject object, Throwable t) {
 		String msg = getLogPrefix()+"received exception ["+t.getClass().getName()+"] from ["+object.getName()+"]";
-		if (ONERROR_CONTINUE.equalsIgnoreCase(getOnError())) {
-//			warn(msg+", will continue processing messages when they arrive: "+ t.getMessage());
-			error(msg+", will continue processing messages when they arrive",t);
-		} else if (ONERROR_RECOVER.equalsIgnoreCase(getOnError())) {
+		if (ON_ERROR.CONTINUE.equals(getOnErrorEnum())) {
```

Comment:
Dit mag als `getOnErrorEnum()==ON_ERROR.CONTINUE`
of beter nog 
`switch (getOnErrorEnum()) {`